mbtiles/scripts/cli.py

```diff
@@ -208,33 +304,28 @@ def mbtiles(ctx, files, output, overwrite, title, description,
         north = min(85.051129, north)
 
         # Initialize iterator over output tiles.
-        tiles = mercantile.tiles(
-            west, south, east, north, range(minzoom, maxzoom + 1))
-
-        for tile, contents in pool.imap_unordered(process_tile, tiles):
-
-            if contents is None:
-                log.info("Tile %r is empty and will be skipped", tile)
-                continue
-
-            # MBTiles have a different origin than Mercantile/tilebelt.
-            tiley = int(math.pow(2, tile.z)) - tile.y - 1
-
-            # Optional image dump.
-            if image_dump:
-                img_name = '%d-%d-%d.%s' % (
-                    tile.x, tiley, tile.z, img_ext)
-                img_path = os.path.join(image_dump, img_name)
-                with open(img_path, 'wb') as img:
-                    img.write(contents)
-
-            # Insert tile into db.
-            cur.execute(
-                "INSERT INTO tiles "
-                "(zoom_level, tile_column, tile_row, tile_data) "
-                "VALUES (?, ?, ?, ?);",
-                (tile.z, tile.x, tiley, sqlite3.Binary(contents)))
-
-            conn.commit()
+        tiles = mercantile.tiles(west, south, east, north, range(minzoom, maxzoom + 1))
+
+        if implementation == "cf":
```

Comment:
I'd be inclined to keep someone from accidentally selecting an implementation that will crash. Perhaps something like:
```
if implementation == 'cf':
    if sys.version_info < (3, 7):
        warnings.warn("Option 'cf' requires python>=3.7; setting implementation to 'mp'")
        ...
```